### PR TITLE
Another solution for name identify

### PR DESCRIPTION
**Pull request by ZTWHHH**
Use Stanford Named Entity Recognizer

### CODE REVIEW

File: ArchiveScraping/name_search.ipynb [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    "import nltk\n",
+    "import pandas as pd\n",
+    "from nltk.tag import StanfordNERTagger\n",
+    "from nltk.tokenize import word_tokenize\n",
+    "\n",
+    "# TO use Stanford Named Entity Recognizer, you need to download file first\n",
+    "# Download website link: https://nlp.stanford.edu/software/CRF-NER.html\n",
+    "\n",
+    "jar = '/Users/ztwhhh/Desktop/stanford-ner-4.0.0/stanford-ner.jar'\n",
+    "model = '/Users/ztwhhh/Desktop/stanford-ner-4.0.0/english.all.3class.distsim.crf.ser.gz'\n",
```
Comment (royceschultz) [inline ArchiveScraping/name_search.ipynb:18]: Can you use relative file paths so it's not system specific? If the model files aren't too large, you might want to commit them.

Comment (ZTWHHH) [reply ArchiveScraping/name_search.ipynb:18]: Yeah, I will upload the model file later，and make them work.
Btw, could we run jupyter notebook directly on github?

Comment (royceschultz) [reply ArchiveScraping/name_search.ipynb:18]: We could use [Googel Colab](https://colab.research.google.com/) but in my experience it doesn't work that great with many users. I think running locally is fine. Git Diffs aren't great for ipynb files though. Use a .py module when possible and import the computational functions into the jupyter notebook.

Comment (ZTWHHH) [reply ArchiveScraping/name_search.ipynb:18]: Got, I will try to make it work. Thanks!

**Pull request closed without merging**